tests/security/apparmor/aa_autodep.pm

```diff
@@ -74,7 +74,8 @@ sub run {
 
     # delete cache file of aa-autodep generated profile, so that the next reload creates a fresh cache of /etc/apparmor.d/usr.sbin.nscd
     # (wouldn't happen without deleting the cache file because the cache timestamp is newer than the profile (+ used abstractions) timestamp)
-    assert_script_run "rm -v /var/cache/apparmor/*/usr.sbin.nscd";
+    my $target_dir = (is_sle('>=15') || is_opensuse) ? '/var/cache/apparmor/' : '/var/lib/apparmor/cache/';
```

Comment:
but in sle-15-sp1 didn't find this path...

Reply:
sleep deprivation is kicking in xD thanks for noticing, fixed